awx_collection/plugins/modules/tower_job_launch.py

```diff
@@ -68,6 +68,19 @@
     job_id: "{{ job.id }}"
     timeout: 120
 
+- name: Launch a job template with Extra Vars on remote tower instance
+  tower_job_launch:
+    job_template: "My Job Template"
+    extra_vars: {
+      var1: "My First Variable",
+      var2: "My Second Variable",
+      var3: "My Third Variable"
+      }
```

Comment:
My preference (to be more consistent with how Ansible playbooks tend to be written, anecdotally), would be to lose the brackets.